patches/server/0010-Adventure.patch

```diff
@@ -977,6 +977,10 @@ index 0000000000000000000000000000000000000000..3dc613116c086444ece88bcb0a569eee
 +    // Component
 +
 +    public static Component asAdventure(final net.minecraft.network.chat.Component component) {
++        if (component instanceof AdventureComponent) {
```

Comment:
You can use instanceof cast, that would simplify the code and make it more readable:

```java
if (component instanceof AdventureComponent aComp) {
           // don't convert to json and back if already backed by an adventure Component
           return aComp.adventure;
}
```

Reply:
This style is not used anywhere else in the class (see line 1030), Idk if this should be changed.

Reply:
Well... some of the class was written before that was a thing. I think we'd prefer to use the newer instanceof where we can.

Reply:
Ok, changed :+1: